tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1062,6 +1062,39 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system",
 	},
 
+	/** Firmware layout for the TL-WA1201 v2 */
+	{
+		.id     = "TL-WA1201-V2",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+		        "{product_name:TL-WA1201,product_ver:2.0.0,special_id:45550000}\n"
+			"{product_name:TL-WA1201,product_ver:2.0.0,special_id:55530000}\n",
+		.part_trail = 0x00,
+		.soft_ver = "soft_ver:1.0.1 Build 20200709 rel.66244\n",
```

Comment:
Does all of it really belong here?

Reply:
I extracted these from the oem firmware

Reply:
Of course you did, but that's not my question. My question is whether we need all the stuff after "1.0.1" ...

Reply:
Ok I can check if I can flash the factory image via the oem interface if there is _no_ `Build` and `rel.` entry when I get around it.

With this exact firmwar header `soft_ver` I verified that flashing from oem interface works so I'm a bit reluctant to change it.

Are there any other places `soft_ver` is referred to / used?